### PR TITLE
add new fields with array of bytes

### PR DESCRIPTION
**Pull request by spengadobe**
[PLATIR-19801](https://jira.corp.adobe.com/browse/PLATIR-19801)

### CODE REVIEW

File: extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
                   "type": "object",
                   "properties": {
                     "https://ns.adobe.com/experience/intelligentServices/sendTimeScoreForClick": {
-                      "title": "Send Time Score for Click",
+                      "title": "Send Time Score for Click (to be deprecated)",
```
Comment (mjasrotia) [inline extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json:35]: Please add "meta:status": "deprecated" to the property to deprecate the property.

File: extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
                       "meta:descriptionId": "profile-journeyai-sendtimeoptimization##https://ns.adobe.com/experience/intelligentServices/sendTimeScoreForClick##description##72351"
                     },
                     "https://ns.adobe.com/experience/intelligentServices/sendTimeScoreForOpen": {
-                      "title": "Send Time Score for Open",
+                      "title": "Send Time Score for Open (to be deprecated)",
```
Comment (mjasrotia) [inline extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json:47]: Please add "meta:status": "deprecated" to the property to deprecate the property.

File: extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json [hunk 3]
```diff
@@ -65,7 +89,7 @@
                   "type": "object",
                   "properties": {
                     "https://ns.adobe.com/experience/intelligentServices/sendTimeScoreForOpen": {
-                      "title": "Send Time Score for Open",
+                      "title": "Send Time Score for Open (to be deprecated)",
```
Comment (mjasrotia) [inline extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json:92]: Please add "meta:status": "deprecated" to the property to deprecate the property.

File: extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json (hunk 1 above)
Comment (spengadobe) [reply extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json:35]: Hi @mjasrotia, as I mentioned in the Jira ticket, we plan to keep these "to be deprecated" fields co-existing with the newly added fields for a while for old data preserving and new score validation. We will mark the fields as deprecated later. Is this allowed?

Comment (prabhum2) [reply extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json:35]: @spengadobe does this mean data for both properties will be populated in the dataset ? if it is just to access old data , you can still access old data set with old structure even when you deprecate these properties , its just that you will not be able to populate new data set using deprecated properties . Hope this makes sense

Comment (spengadobe) [reply extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json:35]: Got it. I will change the code then. Thanks!

**Review by prabhum2 — APPROVED** [review]

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**